src/util/resort.js

```diff
@@ -47,10 +47,6 @@ resort.objectify = function (ad, q) {
         var ml = [];
         ad.forEach(function (d, i) {
             var c = d.Categories;
-            var parent = "Home";
```

Comment:
could you comment on this change?

Reply:
This eliminates an unused variable warning in the development server console.